src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java

```diff
@@ -781,6 +782,44 @@ public void testGetIdentifiablesCount() throws Exception {
         assertEquals(4, identifiablesCount.size());
     }
 
+    @Test
+    public void testGetIdentifiablesDuplicatedCount() throws Exception {
+        UUID filterId1 = UUID.randomUUID();
+        UUID filterId1Duplicated = UUID.randomUUID();
+
+        ArrayList<AbstractExpertRule> rules = new ArrayList<>();
+        createExpertLineRules(rules, new TreeSet<>(Set.of("FR")), new TreeSet<>(Set.of("FR")), new TreeSet<>(Set.of(360., 400.)),
+                new TreeSet<>(Set.of(356.25, 393.7)));
+        CombinatorExpertRule parentRule = CombinatorExpertRule.builder().combinator(CombinatorType.AND).rules(rules).build();
+        ExpertFilter expertFilter = new ExpertFilter(filterId1, new Date(), EquipmentType.LINE, parentRule);
+        ExpertFilter expertFilterDuplicated = new ExpertFilter(filterId1Duplicated, new Date(), EquipmentType.LINE, parentRule);
+
+        insertFilter(filterId1, expertFilter);
+        insertFilter(filterId1Duplicated, expertFilter);
+        checkExpertFilter(filterId1, expertFilter);
+        checkExpertFilter(filterId1Duplicated, expertFilterDuplicated);
+
+        MultiValueMap<String, String> params = new LinkedMultiValueMap<>(Map.of(
+                "networkUuid", List.of(NETWORK_UUID.toString()),
+                "variantId", List.of(VARIANT_ID_1)));
+
+        IdsByGroup content = IdsByGroup.builder().ids(Map.of(
+                "g1", List.of(filterId1, filterId1Duplicated)
+        )).build();
+
+        Map<String, Long> identifiablesCount = objectMapper.readValue(
+                mvc.perform(post(URL_TEMPLATE + "/identifiables-count")
+                                .params(params)
+                                .content(objectMapper.writeValueAsString(content))
+                                .contentType(APPLICATION_JSON))
+                        .andExpect(status().isOk())
+                        .andReturn().getResponse().getContentAsString(),
+                new TypeReference<>() {
+                });
+
+        assertEquals(2, identifiablesCount.get("g1").longValue());
```

Comment:
```suggestion
        IdsByGroup content = IdsByGroup.builder().ids(Map.of(
                "g1", List.of(filterId1, filterId1Duplicated),
                "g2", List.of(filterId1)
        )).build();

        Map<String, Long> identifiablesCount = objectMapper.readValue(
                mvc.perform(post(URL_TEMPLATE + "/identifiables-count")
                                .params(params)
                                .content(objectMapper.writeValueAsString(content))
                                .contentType(APPLICATION_JSON))
                        .andExpect(status().isOk())
                        .andReturn().getResponse().getContentAsString(),
                new TypeReference<>() {
                });

        assertEquals(2, identifiablesCount.get("g1").longValue());
        assertEquals(identifiablesCount.get("g2").longValue(), identifiablesCount.get("g1").longValue());
```

I feel like we could lose this assertion since "2" is a value based on test setup